ArduCopter/autoyaw.cpp

```diff
@@ -330,3 +338,35 @@ AC_AttitudeControl::HeadingCommand Mode::AutoYaw::get_heading()
 
     return heading;
 }
+
+// handle the interface to the weathervane library
+// pilot_yaw can be an angle or a rate or rcin from yaw channel. It just needs to represent a pilot's request to yaw the vehicle to enable pilot overrides.
+#if WEATHERVANE_ENABLED == ENABLED
+void Mode::AutoYaw::update_weathervane(const int16_t pilot_yaw_cds)
+{
+    if (!copter.flightmode->allows_weathervaning()) {
+        return;
+    }
+
+    float yaw_rate_cds;
+    if (copter.g2.weathervane.get_yaw_out(yaw_rate_cds, pilot_yaw_cds, copter.flightmode->get_alt_above_ground_cm()*0.01,
+                                                                       copter.pos_control->get_roll_cd(),
```

Comment:
```suggestion
                                                                       copter.pos_control->get_roll_cd() - copter.attitude_control->get_roll_trim_rad() * 5730.0f,
```
this should remove the roll bias used by heli's.  This is a virtual function in attitude control parent class and is set to zero. So this should have no affect on multicopters

Reply:
Done